libglusterfs/src/event-epoll.c

```diff
@@ -351,7 +351,7 @@ event_register_epoll(struct event_pool *event_pool, int fd,
                      event_handler_t handler, void *data, int poll_in,
                      int poll_out, int notify_poller_death)
 {
-    int idx;
+    int idx = 0;
```

Comment:
It used to be -1 before I broke it with my latest patch that introduced this regression.

Reply:
Set to -1.